packager.go

```diff
@@ -380,7 +388,16 @@ func (this *Packager) ServeHTTP(resp http.ResponseWriter, req *http.Request, par
 		return
 	}
 
-	exchange, err := signedexchange.NewExchange(signURL, http.Header{}, fetchResp.StatusCode, fetchResp.Header, fetchBody)
+	// transform
```

Comment:
Make this a full sentence (or delete if unintended).

Reply:
done